developers/developers.md

```diff
@@ -107,6 +107,14 @@ The behavior of each motor will be defined by its assigned contributions to each
 
 <img src="/images/bluerov-frame.png" class="img-responsive img-center" style="max-height:250px;">
 
+## Troubleshooting
+
+1. If something goes wrong while compiling:
+    1. Check the **Compiling** section to set up your environment.
```

Comment:
hyperlink compiling, but I think that following the instructions on this page is an implicit troubleshooting step.